From bd17e654c2e38a46ea34b6096f108e0385031133 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Fri, 1 May 2020 13:43:26 -0700
Subject: [PATCH 694/830] ANDROID: README: Add requirement for "Bug:" tag for
 FROMLIST
X-NVConfidentiality: public

There needs to be a bug for FROMLIST patches to discuss
rationale for why the patch is so urgent that it needs
to be merged before maintainers accept it upstream.

Bug: 155420414
Signed-off-by: Todd Kjos <tkjos@google.com>
Change-Id: I33bfc9b1edcead7a9b300ba57a2d371a9ea2f74e
---
 README.md | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/README.md b/README.md
index 5612ab828a10..468ac2084b21 100644
--- a/README.md
+++ b/README.md
@@ -111,6 +111,8 @@ must be a stable maintainer branch (not rebased, so don't use `linux-next` for e
 - If the patch has been submitted to LKML, but not accepted into any maintainer tree
     - tag the patch subject with `FROMLIST:`
     - add a `Link:` tag with a link to the submittal on lore.kernel.org
+    - add a `Bug:` tag with the Android bug (required for patches not accepted into
+a maintainer tree)
     - if changes were required, use `BACKPORT: FROMLIST:`
     - Example:
 ```
-- 
2.17.1

